src/gmp/z.d

```diff
@@ -1852,6 +1882,18 @@ unittest
     assert(storage2 == expected);
 }
 
+unittest
+{
+    auto prime = 199.Z;
+    for (int i = 0; i < 20; ++i) {
+        prime = nextPrime(prime);
+        ubyte[] data = prime.to!(ubyte)(WordOrder.mostSignificantWordFirst, Endianess.littleEndian, 0);
```

Comment:
Can you add two more levels of for loops that iterates all possible values of word order and endianess?